<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>594</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1029</width>
    <height>594</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1029</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1029</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>611</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>波形显示区域</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>601</width>
       <height>361</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>流量</string>
      </attribute>
      <widget class="PlotWidget" name="plotView_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>571</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>温度</string>
      </attribute>
      <widget class="PlotWidget" name="plotView_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>571</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>动脉压</string>
      </attribute>
      <widget class="PlotWidget" name="plotView_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>571</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>静脉压</string>
      </attribute>
      <widget class="PlotWidget" name="plotView_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>571</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>新鲜液压力</string>
      </attribute>
      <widget class="PlotWidget" name="plotView_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>571</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>废液压力</string>
      </attribute>
      <widget class="PlotWidget" name="plotView_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>571</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>电导</string>
      </attribute>
      <widget class="PlotWidget" name="plotView_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>571</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>PH</string>
      </attribute>
      <widget class="PlotWidget" name="plotView_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>571</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>409</y>
      <width>781</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>参数设置区域</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>761</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_1" native="true">
        <widget class="QLineEdit" name="input_1">
         <property name="geometry">
          <rect>
           <x>53</x>
           <y>30</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="Button_send1">
         <property name="geometry">
          <rect>
           <x>51</x>
           <y>80</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>16</x>
           <y>9</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>新鲜液和废液(单位：ml/min)</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <widget class="QLineEdit" name="input_2">
         <property name="geometry">
          <rect>
           <x>55</x>
           <y>30</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="Button_send2">
         <property name="geometry">
          <rect>
           <x>53</x>
           <y>80</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>26</x>
           <y>10</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>超滤(单位：ml/min)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <widget class="QLineEdit" name="input_3">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="Button_send3">
         <property name="geometry">
          <rect>
           <x>52</x>
           <y>80</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>43</x>
           <y>10</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>命令</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <widget class="QLineEdit" name="input_4">
         <property name="geometry">
          <rect>
           <x>52</x>
           <y>30</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="Button_send4">
         <property name="geometry">
          <rect>
           <x>51</x>
           <y>80</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>43</x>
           <y>10</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>血泵</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>10</y>
      <width>171</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>17</y>
       <width>163</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>AcadEref</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>串口号</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="Button_quit">
        <property name="font">
         <font>
          <family>AcadEref</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="Button_FindPort">
        <property name="font">
         <font>
          <family>AcadEref</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>检测串口</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>150</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="Box_get_port">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="Button_Rec">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>接收</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="Button_Open">
        <property name="font">
         <font>
          <family>AcadEref</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>打开</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>221</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>数值显示区域</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>511</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLabel" name="label_value_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_name_3">
        <property name="text">
         <string>动脉压：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_value_3">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_value_1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_value_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_name_5">
        <property name="text">
         <string>新鲜液压力：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_value_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_name_7">
        <property name="text">
         <string>电导（ms/cm）：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_name_1">
        <property name="text">
         <string>流量（ml/min）：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_name_6">
        <property name="text">
         <string>废液压力：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_name_2">
        <property name="text">
         <string>温度（℃）：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_value_7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_name_4">
        <property name="text">
         <string>静脉压：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_value_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_name_8">
        <property name="text">
         <string>PH：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_value_8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
